testsuite/features/step_definitions/command_steps.rb

```diff
@@ -445,6 +445,25 @@
   $server.run("kill $(pidof -x fake_ipmi_host.sh)")
 end
 
+When(/^the server starts mocking a Redfish host$/) do
+  $server.run("mkdir -p /root/Redfish-Mockup-Server/")
+  ["redfishMockupServer.py", "rfSsdpServer.py"].each do |file|
```

Comment:
I prefer single quotes if we don't have something that needs to be escaped.

Reply:
done